index.html

```diff
@@ -2,86 +2,49 @@
-				<li><a href="gallery.html">Gallery</a></li>
-			<li><a href="about.html">About Us</a></li>
-			<li><a href="contact.html">Contact Us</a></li>
-			</ul>
-			</div>
-			<br /> <br /> 
-			
-			<div id='divBody'>
-			
-			<div style="display: inline;">
-			<img style="width: 35%; height: 45%; display: inline; padding-right: 1%;" align="left" src="Images/etienne.jpg"/>
-			Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin libero justo, maximus ut velit eget, elementum dapibus odio. 
-			Donec magna lorem, elementum in imperdiet ac, sodales a velit. Donec maximus quis nunc pellentesque imperdiet. Sed in luctus orci, 
-			sed fringilla neque. Maecenas at lacus quis ante consequat molestie. Proin venenatis eu neque et viverra. Nullam felis lacus, 
-			laoreet nec nisi vel, accumsan dignissim purus. Proin non viverra dui. Quisque id urna vestibulum, aliquet nisi sit amet, commodo ante. 
-			Nullam bibendum odio metus, ac commodo elit consectetur ac. 	
-			
-			</div>
+		<header>
+			<script>
+				$("header").load("banner.html")
+			</script>
+		</header>
+		<div id="body">
+			<img class="photo" src="Images/etienne.jpg"/>
+			<p>
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin libero justo, maximus ut velit eget, elementum dapibus odio. 
+				Donec magna lorem, elementum in imperdiet ac, sodales a velit. Donec maximus quis nunc pellentesque imperdiet. Sed in luctus orci, 
+				sed fringilla neque. Maecenas at lacus quis ante consequat molestie. Proin venenatis eu neque et viverra. Nullam felis lacus, 
+				laoreet nec nisi vel, accumsan dignissim purus. Proin non viverra dui. Quisque id urna vestibulum, aliquet nisi sit amet, commodo ante. 
+				Nullam bibendum odio metus, ac commodo elit consectetur ac.
+			</p>
 			<p>
-			Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin libero justo, maximus ut velit eget, elementum dapibus odio. 
-			Donec magna lorem, elementum in imperdiet ac, sodales a velit. Donec maximus quis nunc pellentesque imperdiet. Sed in luctus orci, 
-			sed fringilla neque. Maecenas at lacus quis ante consequat molestie. Proin venenatis eu neque et viverra. Nullam felis lacus, 
-			laoreet nec nisi vel, accumsan dignissim purus. Proin non viverra dui. Quisque id urna vestibulum, aliquet nisi sit amet, commodo ante. 
-			Nullam bibendum odio metus, ac commodo elit consectetur ac. 	
+				Lorem ipsum dolor sit amet, consectetur adipiscing elit. Proin libero justo, maximus ut velit eget, elementum dapibus odio. 
+				Donec magna lorem, elementum in imperdiet ac, sodales a velit. Donec maximus quis nunc pellentesque imperdiet. Sed in luctus orci, 
+				sed fringilla neque. Maecenas at lacus quis ante consequat molestie. Proin venenatis eu neque et viverra. Nullam felis lacus, 
+				laoreet nec nisi vel, accumsan dignissim purus. Proin non viverra dui. Quisque id urna vestibulum, aliquet nisi sit amet, commodo ante. 
+				Nullam bibendum odio metus, ac commodo elit consectetur ac. 	
```

Comment:
It bothers me to no end when nerds fight about 4 spaces vs tab, but it bothers me more when you don't use one or the other. I converted your spaces into tabs. I'm more of a 4 spaces guy myself, but you were mostly tabs everywhere else, so I wanted to keep it consistent.